src/test/Unit/auth.test.js

```diff
@@ -0,0 +1,58 @@
+import { login } from "../../js/api/auth/login.js";
+import { logout } from "../../js/api/auth/logout.js";
+const loginCredentials = {
+  email: "stian.busengdal@stud.noroff.no",
+  password: "stian123123",
```

Comment:
If these are real details, they should be in an .env file so they don't leak to the public